fs.go

```diff
@@ -399,8 +399,8 @@ func (fs *TorrentFS) SeedingLocal(ctx context.Context, filePath string, isLinkMo
 	// 4. copy or link, will not cover if dst exist!
 	ih := common.Address(mi.HashInfoBytes())
 	log.Info("Local file Seeding", "ih", ih.Hex(), "path", dataPath)
-	linkDst := strings.TrimPrefix(strings.ToLower(ih.Hex()), common.Prefix)
-	linkDst = filepath.Join(fs.storage().TmpDataDir, linkDst)
+	ihHexLower := strings.TrimPrefix(strings.ToLower(ih.Hex()), common.Prefix)
```

Comment:
```infoHash = strings.TrimPrefix(strings.ToLower(ih.Hex()), common.Prefix)``` is it OK to set ih directly here ?

Reply:
Definitely, it's an unnecessary variable initialize here.